tests/api/v1/samples/upsert.js

```diff
@@ -505,38 +505,4 @@ describe(`api: POST ${path}`, () => {
       });
     });
   });
-
-  describe('on case insensitive upsert', () => {
-    beforeEach((done) => {
-      Sample.create({
-        name: `${subject.absolutePath}|${aspect.name}`,
-        value: '1',
-        aspectId: aspect.id,
-        subjectId: subject.id,
-      })
-      .then(() => api.post(path)
-      .set('Authorization', token)
-      .send({
-        // updates the name to use lowercase
-        name: `${subject.absolutePath}|${aspect.name}`.toLowerCase(),
-        value: '2',
-      }))
-      .then(() => done())
-      .catch(done);
-    });
-
-    it('existing sample is not duplicated', (done) => {
-      api.get('/v1/samples?name=' + `${subject.absolutePath}|${aspect.name}`)
-      .end((err, res) => {
-        if (err) {
-          done(err);
-        }
-
-        expect(res.body).to.have.length(1);
-        expect(res.body[0].name)
-        .to.equal(`${subject.absolutePath}|${aspect.name}`.toLowerCase());
```

Comment:
remove the test that has incorrect expectations: sample name should NOT be updated